src/const.js

```diff
@@ -0,0 +1,17 @@
+export const ActivityTypeEnum = {
+  BOOK: 0,
+  PEOPLE: 1,
+  CONFERENCE: 2,
+}
+
+export const PriorityEnum = {
+  LOW: 0,
+  MEDIUM: 1,
+  HIGH: 2,
+}
+
+export const StatusEnum = {
+  TODO: 0,
+  DOING: 1,
```

Comment:
Is DOING useful ? Or is it a too advanced functionality for our TODO system ? (I'm more thinking in term of UX than the functionality itself)